### ISSUE TITLE
Switch and button sizes not matched. :C

### ISSUE THREAD

**Issue by Sadnessa** [epicmaxco/vuestic-ui#1323]
**Vuestic-ui version:** 1.3.1

**Button and switch should have same size in pixels**

**What is the current behavior?**
I used default size property for button and switch, but as you can see, actual sizes are different.
![изображение](https://user-images.githubusercontent.com/67637329/145320429-7e993dc4-1b78-4c88-8378-9b11b3c56bec.png)
![изображение](https://user-images.githubusercontent.com/67637329/145320916-5aa34865-259f-4fd8-a1e9-cc2122b90d29.png)
![изображение](https://user-images.githubusercontent.com/67637329/145320945-273bf1a3-0672-4aa3-9ddb-6c875b9300cc.png)

**Comment by asvae** [comment]
Hey.

That's not something we pursued (so not a bug), but would be interesting to look if we can unify styling for switch and button so they can be used together with better harmony.

Thanks.

**Comment by asvae** [comment]
@xx13 would you consider this interesting?

**Comment by asvae** [comment]
@aluarius would be great if you can check this problem still exists.

**Comment by xx13** [comment]
@asvae yep, buttons and inputs are synced. need to review other elements, like toggles

**Comment by aluarius** [comment]
Yes, this problem still exists.

![](http://joxi.ru/xAeZ47ZCMekdBr.jpg)

But in my opinion it can wait until switcher (and other not matching components) rework via new design system.
